en/advanced_config/tuning_the_ecl_ekf.md

```diff
@@ -126,6 +127,28 @@ The [GPS_2_CONFIG](../advanced_config/parameter_reference.md#GPS_2_CONFIG) param
 * Where receivers output at different rates, the blended output will be at the rate of slower receiver. 
   Where possible receivers should be configured to output at the same rate.
 
+
+#### GPS Performance Requirements
+
+For the ECL to accept GPS data for navigation, certain minimum requirements need to be satisfied over a period of 10 seconds (minimums are defined in the [EKF2_REQ_*](../advanced_config/parameter_reference.md#EKF2_REQ_EPH) parameters)
+
+The table below shows the different metrics directly reported or calculated from the GPS data, and the minimum required values for the data to be used by ECL. 
+In addition, the *Average Value* column shows typical values that might reasonably be obtained from a standard GNSS module (e.g. uBlox M8 series) - i.e. values that are considered good/acceptable.
+
+Metric | Minimum required | Average Value | Units | Notes
+--- | --- | --- | --- | ---
+eph | 3&nbsp;([EKF2_REQ_EPH](../advanced_config/parameter_reference.md#EKF2_REQ_EPH)) | 0.8 | m | Standard deviation of horizontal position error
+epv | 5&nbsp;([EKF2_REQ_EPV](../advanced_config/parameter_reference.md#EKF2_REQ_EPV)) | 1.5 | m | Standard deviation of vertical position error
+Number of satellites | 6&nbsp;([EKF2_REQ_NSATS](../advanced_config/parameter_reference.md#EKF2_REQ_NSATS)) | 14 | - | 
+Speed variance | 0.5 | 0.3 | m/s | 
+Fix type | 3 | 4 | - | 
+hpos_drift_rate | 0.1&nbsp;([EKF2_REQ_HDRIFT](../advanced_config/parameter_reference.md#EKF2_REQ_HDRIFT)) | 0.01 | m/s | Drift rate calculated from reported GPS position (when stationary).
+vpos_drift_rate | 0.2&nbsp;([EKF2_REQ_VDRIFT](../advanced_config/parameter_reference.md#EKF2_REQ_VDRIFT)) | 0.02 | m/s | Drift rate calculated from reported GPS altitude (when stationary).
+hspd | 0.1&nbsp;([EKF2_REQ_SACC](../advanced_config/parameter_reference.md#EKF2_REQ_SACC)) | 0.01 | m/s | Filtered magnitude of reported GPS horizontal velocity.
+
+> **Note** The `hpos_drift_rate`, `vpos_drift_rate` and `hspd` are calculated over a period of 10 seconds and published in the `ekf2_gps_drift` topic.
```

Comment:
@hamishwillee Might be worth mentioning that ekf2_gps_drift topic is not logged anymore.
See [here](https://github.com/PX4/Firmware/pull/11813/files)

Reply:
Thanks very much. I just added "Note that `ekf2_gps_drift` is not logged." at the end of this note.